### PR TITLE
Fix dir

### PR DESCRIPTION
**Pull request by xujiangyu**
fix:
 1、os.makedir ➡️ os.mkidir
 2、xgb(plaintext) evaluation raise error

### CODE REVIEW

**Review by hobo0cn — CHANGES_REQUESTED** [review]
Please follow [PR checklist](https://github.com/primihub/primihub/blob/develop/.github/PULL_REQUEST_TEMPLATE.md). @xujiangyu

File: python/primihub/context.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ def mk_output_dir(output_dir):
         if output_dir:
             if not os.path.exists(output_dir):
                 try:
-                    os.makedir(output_dir)
+                    #os.makedir(output_dir)
```
Comment (LinuxSuRen) [inline python/primihub/context.py:106]: Forget to delete this line?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: python/primihub/context.py (hunk 1 above)
Comment (xujiangyu) [reply python/primihub/context.py:106]: ok,

**Comment by xujiangyu** [comment]
I will submit it later.

**Pull request closed without merging**